### PR TITLE
Error failed: Success (!) in logs

### PR DESCRIPTION
**Pull request by woody-apple**
Logs show this constantly, which makes it hard to parse for just errors:

* Error                  | 2024-04-19 07:18:15.028262-0700 | com.csa.matter                 | Inet                           | SendMessage() to UDP:[fd55:7e52:e56f:1:97a5:a2d0:34fb:8f1b]:5540 failed: src/inet/UDPEndPoint.cpp:121: Success
* Error                  | 2024-04-19 07:18:15.449594-0700 | com.csa.matter                 | Inet                           | SendMessage() to UDP:[fd55:7e52:e56f:1:86a1:4c3f:e4cf:4332]:5540 failed: src/inet/UDPEndPoint.cpp:121: Success
* Error                  | 2024-04-19 07:18:16.003170-0700 | com.csa.matter                 | Inet                           | SendMessage() to UDP:[fd55:7e52:e56f:1:97a5:a2d0:34fb:8f1b]:5540 failed: src/inet/UDPEndPoint.cpp:121: Success

### CODE REVIEW

**Comment by woody-apple** [comment]
Note to reviewers, this doesn't build:

```
../../src/transport/SessionManager.cpp:481:14: error: 'IsSuccess' was not declared in this scope
  481 |         if (!IsSuccess(err))
      |              ^~~~~~~~~
```

That said, I am not confident this fixes it, if anyone has a better suggestion for why the log above is appearing, would love to know what's wrong.

File: src/transport/SessionManager.cpp [hunk 1]
```diff
@@ -478,7 +478,7 @@ CHIP_ERROR SessionManager::SendPreparedMessage(const SessionHandle & sessionHand
     {
         CHIP_ERROR err = mTransportMgr->SendMessage(*destination, std::move(msgBuf));
 #if CHIP_ERROR_LOGGING
-        if (err != CHIP_NO_ERROR)
```
Comment (ksperling-apple) [inline src/transport/SessionManager.cpp:481]: The existing check works fine, you must have been running a build from the brief time between when the logging was introduced and when the `err != CHIP_NO_ERROR` check was added.

**Comment by woody-apple** [comment]
No longer needed, thanks @ksperling-apple !
